copilot_proxy/utils/codegen.py

```diff
@@ -72,14 +74,17 @@ def to_word_list_format(word_dict, tokenizer):
     def generate(self, data):
         model_name = "fastertransformer"
         prompt = data['prompt']
+        max_tokens = data.get('max_tokens', 16)
+        encoded_tokens = self.tokenizer.encode(prompt).ids
+        if max_tokens + len(encoded_tokens) > self.MAX_MODEL_LEN:
+            print("Max tokens + prompt length exceeds maximum model length, shortening prompt")
```

Comment:
Is using `print` here intended? Or is there some logging facility used elsewhere that would be better?